.travis.yml

```diff
@@ -3,7 +3,8 @@ language: python
 services:
   - docker
 python:
-  - "3.6"
+  - "3.7"
```

Comment:
don't you need 3.9 also, as you added it to travis.yml ?

Reply:
Have it in tox yes.